frontend/src/ui-components/AcmCharts/AcmDonutChart/AcmDonutChart.tsx

```diff
@@ -26,7 +26,7 @@ type LegendData = {
 /* istanbul ignore next */
 const getStyles = (props: StyleProps) => ({
   card: css({
-    maxHeight: '259px',
+    maxHeight: '300px',
```

Comment:
Why this chanaged?

Reply:
You're right, I changed this previously when I was trying to add a footer to the card. I'll have it reverted.